publish_upm.sh

```diff
@@ -1,6 +1,10 @@
-#!/bin/sh
-git subtree split --prefix=Assets/Mirror -b upm
+#!/bin/sh -e
+
+BRANCH=$1
+TAG=$2
+git subtree split --prefix=Assets/Mirror -b $BRANCH
 #git filter-branch --prune-empty --tree-filter 'rm -rf Tests' upm
-git filter-repo --invert-paths --path Tests --refs upm
-git tag $1 upm
-git push -u origin upm --tags
+git gc
+git filter-repo --force --invert-paths --path Tests --path Tests.meta --path Examples.meta --path-rename "Examples:Samples~" --refs $BRANCH
+git tag $TAG $BRANCH
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Double quote to prevent globbing and word splitting.](https://app.codacy.com/manual/paulpach/MirrorNG/pullRequest?prid=4804713)